draft-ietf-tsvwg-udp-ecn.md

```diff
@@ -229,8 +233,8 @@ An example of the technique described above can be found at {{CHROMIUM-POSIX}}.
 
 ### Windows
 
-The author did not experiment with setting a windows socket to send an ECN
-mark.
+At the time of this writting, Windows does not provide a way to configure a
```

Comment:
```suggestion
At the time of this writting, Windows does not provide a way to configure
```

Not a native speaker. Please feel free to ignore.

Reply:
Not a native speaker easier :) I am on the fence about which version is the most correct, so I trust you on that one.